lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,30 @@
+#Time complexity: O(n+m+l)
+#Space complexity: O(n^2)
 
 def palindrome_permutation?(string)
```

Comment:
The time complexity is O(n^2) because `unshift` has O(n) and you're doing it n times.  The space complexity O(n) as well. 

You seem to be trying to check to see if the string is a palindrome.   The problem states that you are checking to see if the letters could be re-arranged into a palindrome.

Reply:
Ah...I went back to see the problem. I misunderstood the problem. My mind was thinking about the palindrome that I did a few weeks ago.